### PR TITLE
[eos]: Fix Sanity issues

### LINKED ISSUE #73: Fix ansible sanity test ignored failures

### PR DESCRIPTION
**Pull request by GomathiselviS**
##### SUMMARY
Fixes #73 

Remove ignore_txt by fixing the sanity issues

### CODE REVIEW

**Comment by GomathiselviS** [comment]
recheck

**Comment by pabelanger** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by trishnaguha** [comment]
@GomathiselviS Can you give an update on this PR?

File: plugins/modules/eos_l2_interface.py [hunk 1]
```diff
@@ -367,7 +411,7 @@ def main():
     """ main entry point for module execution
     """
     element_spec = dict(
-        name=dict(type="str", aliases=["interface"]),
+        name=dict(type="str", aliases=["interface"], required=True),
```
Comment (rohitthakur2590) [inline plugins/modules/eos_l2_interface.py:414]: top-level name cannot be required for modules having the aggregate key

**Comment by GomathiselviS** [comment]
> @GomathiselviS Can you give an update on this PR?

@trishnaguha Waiting for a +1 so that I can add the gate label.

**Pull request merged**